Add uimuldw function (#219)
stdlib-js/stdlib: lib/node_modules/@stdlib/math/base/special/uimuldw/docs/repl.txt

{{alias}}( [out,] a, b )

>     Multiplies two unsigned 32-bit integers and returns an array

Reviewer: This description needs to be re-aligned with 80 char width. I believe `of two` can be moved to this line, which may affect subsequent lines.

>     Perform multiplication of two unsigned 32-bit integers and return an array

Reviewer: ```suggestion
    Multiplies two unsigned 32-bit integers and returns an array
```

>     of two unsigned 32-bit integers which represents their 64-bit unsigned

Reviewer: ```suggestion
    of two unsigned 32-bit integers which represents the 64-bit unsigned
```

    integer product.

    Parameters
    ----------

>     out: Array (optional)

Reviewer: ```suggestion
    out: ArrayLikeObject (optional)
```

        The output array.

    a: integer
        Unsigned 32-bit integer.

    b: integer
        Unsigned 32-bit integer.

    Returns
    -------

>     out: Array

Reviewer: ```suggestion
    out: ArrayLikeObject
```
